configuration/exodus-pipeline.yaml

```diff
@@ -0,0 +1,297 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Description: Configuration for exodus-cdn CodePipeline
+
+Parameters:
+  env:
+    Type: String
+    AllowedValues:
+      - dev
+      - stage
+      - prod
+    Default: dev
+    Description: The environment in which to deploy functions
+  oai:
+    Type: String
+    Description: The origin access identity ID associated with the environment
+  github-token:
+    Type: String
+    Default: None
+    Description: A GitHub access token token to authorize
+  region:
+    Type: String
+    Default: us-east-1
+    Description: The region in which resources are established
+  email:
+    Type: String
+    Default: project-exodus@redhat.com
+    Description: The email address to which notifications are sent
+
+Mappings:
+  SourceMap:
+    stage:
+      object-key: build-dev/exodus-lambda-pkg.yaml
+    prod:
+      object-key: build-stage/exodus-lambda-pkg.yaml
+
+Resources:
+  SNSTopic:
+    Type: AWS::SNS::Topic
+    Properties:
+      TopicName: !Sub exodus-sns-${env}
+      Subscription:
+        - Endpoint: !Ref email
+          Protocol: email
+
+  CodePipelineNotification:
+    Type: AWS::CodeStarNotifications::NotificationRule
+    Properties:
+      Name: !Sub exodus-pipeline-${env}
+      DetailType: FULL
+      EventTypeIds:
+        - codepipeline-pipeline-pipeline-execution-failed
+      Resource: !GetAtt CodePipeline.Arn
+      Targets:
+        - TargetType: SNS
+          TargetAddress: !GetAtt SNSTopic.Arn
+
+  CodePipeline:
+    Type: AWS::CodePipeline::Pipeline
+    Properties:
+      Name: !Sub exodus-pipeline-${env}
+      ArtifactStore:
+        Type: S3
+        Location: !Ref CodePipelineBucket
+      RoleArn: !GetAtt CodePipelineRole.Arn
+      Stages:
+        - !If
+          # Create Github source for dev pipeline
+          - !Equals [!Ref env, dev]
+          - Name: Source
+            Actions:
+              - Name: Source
+                ActionTypeId:
+                  Category: Source
+                  Owner: ThirdParty
+                  Provider: Github
+                  Version: 1
+                Configuration:
+                  Owner: release-engineering
+                  Repo: exodus-lambda
+                  Branch: deploy
+                  OAuthToken: !Ref github-token
+                  PollForSourceChanges: false
+                OutputArtifacts:
+                  - Name: SourceArtifact
+          # Create S3 source for stage, prod pipelines
```

Comment:
Is this comment accurate?  The next branch only seems to be entered if env is prod, not stage.

Reply:
Good catch. I originally had stage and prod source stages separated and forgot to change the condition when consolidating.